src/algebra/order/monoid.lean

```diff
@@ -310,7 +310,8 @@ end with_zero
 @[protect_proj, ancestor ordered_add_comm_monoid has_bot]
 class canonically_ordered_add_monoid (α : Type*) extends ordered_add_comm_monoid α, has_bot α :=
 (bot_le : ∀ x : α, ⊥ ≤ x)
-(le_iff_exists_add : ∀ a b : α, a ≤ b ↔ ∃ c, b = a + c)
+(exists_add_of_le : ∀ {a b : α}, a ≤ b → ∃ c, b = a + c)
+(le_add : ∀ a b : α, a ≤ a + b)
```

Comment:
Do we already have a name for this along the lines of `le_add_left`? It would be nice to match that name if so

Reply:
It's `le_self_add` or `self_le_add_left`.

Reply:
I'd suggest naming the field after one of those then, unless the existing name matches a field in another order class